test/strings/basic.jl

```diff
@@ -151,14 +151,6 @@ end
 @test lcfirst("")==""
 @test lcfirst("*")=="*"
 
-#more String tests
-@test convert(String, UInt8[32,107,75], "*") == " kK"
```

Comment:
The third argument was a replacement string for invalid UTF-8 data. This is not a way you're supposed to be able to call `convert`. I'll have to add a deprecation for it before this PR is done.